stable/etcd-operator/templates/operator-deployment.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.deployments.etcdOperator }}
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "etcd-operator.fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: {{ template "etcd-operator.fullname" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    component: "{{ .Release.Name }}-{{ .Values.component }}"
```

Comment:
Charts usually have component labels when they indeed have multiple components which doesn't seem to be the case here. What's the reason for this label? Also, components usually are hard-coded, not configurable. I'd suggest you remove it if possible.